### PR TITLE
fix(avatar): image always covers circle area

### LINKED ISSUE #480: avatar: image should cover entire area
### Expected behavior

Image should cover 100% width OR 100% height, proportionally based on aspect ratio.

### Actual behavior

image

### Steps to repro

Replace avatar image source to an image such as this one: `https://akns-images.eonline.com/eol_images/Entire_Site/2010330/293.WillFerrell.tg.043010.jpg`

### PR DESCRIPTION
**Pull request by jaketrent**
Fixes #480

### CODE REVIEW

File: packages/avatar/src/css/index.js [hunk 1]
```diff
@@ -41,8 +41,9 @@ export default {
     position: 'absolute',
     top: '0',
     left: '0',
-    width: 'auto',
-    height: '100%'
+    width: '100%',
+    height: '100%',
+    objectFit: 'cover'
```
Comment (danethurber) [inline packages/avatar/src/css/index.js]: 👍

**Comment by jaketrent** [comment]
just fyi, object-fit not supported in IE.  Experience there now will be a stretched image.

**Comment by danethurber** [comment]
if that is an issue we can always explore background image and background size

**Comment by jaketrent** [comment]
Not an issue in my mind.  At least not yet...

**Pull request merged**